src/components/App.jsx

```diff
@@ -21,36 +21,17 @@ class App extends React.Component {
           end: new Date(2021, 3, 3),
         },
       ],
-      canEdit: false
     };
-    this.editAvailablity = this.editAvailablity.bind(this);
-    this.updateAvailability = this.updateAvailability.bind(this);
-  }
-
-  updateAvailability() {
-    this.setState({ eventsShowing: [], canEdit: true })
-  }
-
-  editAvailablity({ start, end }) {
-    const { eventsShowing } = this.state;
-    const tempArr = eventsShowing.slice();
-
-    tempArr.push({start, end, title: 'Freetime' });
-
-    this.setState({
-      eventsShowing: tempArr,
-    });
   }
 
   render() {
-    const { eventsShowing, canEdit } = this.state;
+    const app = this;
```

Comment:
Is this syntax for the calendar? I've just never seen it before, looks interesting!